rfc/2022/2022-07-14 - Support Slack-bot auto answer.md

```diff
@@ -26,23 +26,28 @@ There is also an issue where users will post in the vfs-platform-support slack c
 
 The proposed workflow is as follows:
 1. The platform user submits a ticket through the Platform Support Slack-bot
-2. The application filters through the documentation mappings based on the tickets message, using keywords, or topic
-3. If there is a documentation mapping, the application will send back a threaded response to the ticket listing the documentation mappings
-4. The threaded response will contain a "Did this help?" Yes/No question for determining the usefulness of documentation mappings
+3. The application could filter through a record of previous solutions, if available.
+4. The application filters through the documentation mappings based on the tickets message, using keywords, or topic.
+5. If there is a documentation mapping, the application will send back a threaded response to the ticket listing the documentation mappings
+6. The threaded response will contain a promt to review documentation mapping and a "Do these help?" Yes/No question for determining the usefulness of documentation mappings
+7. A "Yes" response closes the support request, while a "No" response proceeds to assign support staff.  
```

Comment:
This is the workflow that Tim Wright recommended also. My only concern with this is the example that you provided for PRs. The documentation we have for PRs is how to create them and the process around getting them approved. If someone wanted a quick review they would have to go through the workflow even though its not helpful (most people who pick the "PR Review" topic are looking for a review and not help on creating one).

The solutions to this are keep the workflow as is (the user doesn't have to indicate if it was helpful before it is assigned) or remove the PR Review answer mappings. If we do the later, we may want to add a topic around PR creation and process to return those mappings.

Reply:
@powellkerry 
You could also split PR Review into two separate options PRs ( actual questions and issues with the PR process) and PR Reviews ( which the application would not touch) would be exclusive to people asking for code reviews.